apps/arweave/include/ar.hrl

```diff
@@ -310,6 +310,23 @@
 %% Maximum size of a single data chunk, in bytes.
 -define(DATA_CHUNK_SIZE, (256 * 1024)).
 
+%% The maximum allowed packing difficulty.
+-define(MAX_PACKING_DIFFICULTY, 32).
+
+%% The number of sub-chunks in a compositely packed chunk with the packing difficulty
+%% between 1 and 32 incl. The composite packing with the packing difficulty 1 matches
+%% approximately the non-composite 2.6 packing in terms of computational costs.
+-define(PACKING_DIFFICULTY_ONE_SUB_CHUNK_COUNT, 32).
```

Comment:
The number of sub-chunks is the same for every packing difficulty, right? i.e. this isn't specifically about "Packing Difficulty 1" right? Or have I misunderstood?

I.e. could this be called `COMPOSITE_PACKING_SUB_CHUNK_COUNT` or something and still be accurate?

Reply:
Hm, okay I see elsewhere we use the term "atomic sub-chunk" or "basic atomic sub-chunk" - are those different from a "packing difficulty 1 sub chunk"?

if they're all the same thing, what do you think about just labeling them all "sub chunk" and dropping the qualifiers - I don't think we have the concept of a sub-chunk anywhere else in the code yet, right?